pytorch_lightning/accelerators/__init__.py

```diff
@@ -11,15 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from pytorch_lightning.accelerators.cpu_accelerator import CPUAccelerator
-from pytorch_lightning.accelerators.ddp2_accelerator import DDP2Accelerator
-from pytorch_lightning.accelerators.ddp_accelerator import DDPAccelerator
-from pytorch_lightning.accelerators.ddp_spawn_accelerator import DDPSpawnAccelerator
-from pytorch_lightning.accelerators.ddp_cpu_spawn_accelerator import DDPCPUSpawnAccelerator
-from pytorch_lightning.accelerators.dp_accelerator import DataParallelAccelerator
-from pytorch_lightning.accelerators.gpu_accelerator import GPUAccelerator
-from pytorch_lightning.accelerators.tpu_accelerator import TPUAccelerator
-from pytorch_lightning.accelerators.horovod_accelerator import HorovodAccelerator
-from pytorch_lightning.accelerators.ddp_hpc_accelerator import DDPHPCAccelerator
-from pytorch_lightning.accelerators.ddp_cpu_hpc_accelerator import DDPCPUHPCAccelerator
-from pytorch_lightning.accelerators.accelerator import Accelerator
+from pytorch_lightning.accelerators.cpu_accelerator import CPUAccelerator  # noqa: F401
```

Comment:
Just out of curiosity, why does the linter complain?

Reply:
Because we do not use import from inits internally (to avoid cyclic imports), they are mostly for end user...